### PR TITLE
Flask Task 3: Add hobbies page

### PR DESCRIPTION
**Pull request by gibbonhug**
[Flask task 3](https://github.com/MAMV3x3/MLH-Fellowship-pe-portfolio-site/issues/12)

I have added a /hobbies route which displays each team member's hobbies along with an image. Member hobby data is stored in new JSON file. Jinja templating loops through the JSON and displays the data on the page. I added basic styling to the page, using flexboxes to align everything.

hobbies

### CODE REVIEW

**Review by MAMV3x3 — APPROVED** [review]
Awesome job, just make sure to resolve the conflicts, otherwise LGTM! 🚀

**Review by a-brown-code — APPROVED** [review]
Looks great! 💯

File: app/__init__.py [hunk 1]
```diff
@@ -84,6 +84,13 @@ def load_member_locations():
         members = data['members']
     return members
 
+def load_member_hobby_data():
+    hobby_file_path = Path(__file__).resolve().parent / 'static' / 'data' / 'hobbies.json'
+    
+    with open(hobby_file_path, 'r') as f:
```
Comment (Pradyuman7) [inline app/__init__.py:90]: Nit: It is always good to handle exceptions/errors like FileNotFoundError in this case.

Although, it should be fine for now, but it's a good thing to keep in mind

**Review by Pradyuman7 — APPROVED** [review]
LGTM 👍

**Pull request merged**